### PR TITLE
Create academics-grants-round page

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Draft grants page

Preview deploy:
https://ethereumorgwebsitedev01-academicsgrantsround.gtsb.io/en/community/grants/academics-grants-round

**Note: we need this page to be live first:** 
https://esp.ethereum.foundation/academics-grants

## Related Issue

None

### CODE REVIEW

**Review by corwintines — COMMENTED** [review]
Just wondering if this should be linked out to somewhere on the site, or if it is just a page we are having that can be sent out to people.

**Comment by wackerow** [comment]
Do we know when the https://esp.ethereum.foundation/academics-grants page will be live? Still getting 404 on this page... Would suggest we leave this in draft to prevent merging before that link is live.

**Comment by samajammin** [comment]
> Just wondering if this should be linked out to somewhere on the site, or if it is just a page we are having that can be sent out to people.

Good call, added a link from the /community/grants/ page 👍 

> Do we know when the https://esp.ethereum.foundation/academics-grants page will be live? Still getting 404 on this page... Would suggest we leave this in draft to prevent merging before that link is live.

Should be live by tomorrow but agreed, just updated this to a draft PR 👍

File: src/content/community/grants/academics-grants-round/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+<ExpandableCard
+	contentPreview="Formal research output which is open source."
+	title="What should the output be?">
+
+   <p>We are aiming to receive academic and formal research outputs, be it a research paper, an experimental report or some sort of comprehensive research output. Whatever the output, the information should be open-sourced and available for the broader community to use. The intellectual property can still be owned by the research team, but it is key that the output is open source.</p>
+
+</ExpandableCard>
+
+<ExpandableCard
+contentPreview="Up to $750.000 in total"
```
Comment (minimalsm) [inline src/content/community/grants/academics-grants-round/index.md:114]:
```suggestion
contentPreview="Up to $750,000 in total"
```

File: src/content/community/grants/academics-grants-round/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+title="What is the budget available for this round?">
+
+   <p>This grants round has up to $750.000 in total, which is to be distributed among selected projects. </p>
```
Comment (minimalsm) [inline src/content/community/grants/academics-grants-round/index.md:117]:
```suggestion
   <p>This grants round has up to $750,000 in total, which is to be distributed among selected projects. </p>
```

File: src/content/community/grants/academics-grants-round/index.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+sidebarDepth: 2
+---
+
+# Academics Grants Round {#academics-grants-round}
+
+<InfoBanner>
+  This round is now open - apply before April 22, 2022! <a href="https://esp.ethereum.foundation/academics-grants/" target="_blank">Apply here</a>.
+</InfoBanner>
+
+_The Ethereum Foundation is sponsoring a wave of grants to support academic work related to Ethereum. This grants round has up to $750.000 in total available. Proposals are due <b>April 22, 2022</b>. Here are all the details you need._
```
Comment (minimalsm) [inline src/content/community/grants/academics-grants-round/index.md:15]:
```suggestion
_The Ethereum Foundation is sponsoring a wave of grants to support academic work related to Ethereum. This grants round has up to $750,000 in total available. Proposals are due <b>April 22, 2022</b>. Here are all the details you need._
```

**Review by minimalsm — COMMENTED** [review]
This should use $750,000 instead of $750.000 if we're denominating USD

**Review by pettinarip — APPROVED** [review]

**Pull request merged**